cassdegrees/templates/list.html

```diff
@@ -49,14 +49,15 @@
     {# Generates table instances for each table in 'data', as well as the appropriate settings to keep it hidden #}
     {% for title, content in data.items %}
         <div id="{{ title }}Table" hidden>
-            <form id="{{ title }}Sorter" method="post">
+            <form method="post">
                 {% csrf_token %}
                 {# Allow manage_courses() function to behave appropriately based on origin of request. The post request can either come from list.html or managecourses.html #}
                 <input type="hidden" id="perform_function" name="perform_function" value="retrieve view from selected">
                 <p class="left">
                     <input class="btn-uni-grad btn-medium" type ="submit" formmethod="get" formaction="{{ render.admin_url_prefix }}create/{% if title == 'Program' %}program{% elif title == 'Subplan' %}subplan{% elif title == 'Course' %}course{% endif %}/" value="+ New {{ title }}">
```

Comment:
Almost wonder if its a better idea here to use list.js search while we are doing all that (like what is being done on the student frontend) to avoid having search logic on the server.

Reply:
I will try to improve the search on the student planner to see if it can get good enough to replace Jack's search functionality.